docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -287,7 +287,7 @@ public class GreetingResource {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     @Path("/greeting/{name}")
-    public String greeting(@PathParam String name) {
+    public String greeting(@PathParam("name") String name) {
```

Comment:
Nope. We are using `org.jboss.resteasy.annotations.jaxrs.PathParam`, not the JAX-RS `@PathParam` and the RESTEasy one doesn't require the name.

And if something needs fixing, it would be making sure the imports are always there so that's it's clear. Typically, in this case, the imports are there.

Reply:
they are almost present everywhere, I will add the missing ones.